### PR TITLE
Merges all fixes into 

### PR DESCRIPTION
**Pull request by NicholasLancey**
## Description

This pull request contains various fixes, as well as the 2 open pull requests all merged into one. Over the past couple months the library has been tested extensively and finally has approval from QA.

### CODE REVIEW

File: src/wtc-perspective-card.js [hunk 1]
```diff
@@ -74,8 +74,12 @@ class PerspectiveCard {
     );
     this.shine = this.element.querySelector(".perspective-card__shine");
 
+    this.cardImage = this.element.querySelector(".card__img--front");
```
Comment (andrewrubin) [inline src/wtc-perspective-card.js]: Looks like this `cardImage` isn't used anywhere. Should we remove, to avoid confusion?

**Review by andrewrubin — CHANGES_REQUESTED** [review]
Code all looks good to me!

Let's make sure we bump to a minor version before merging, via `npm version minor`

File: src/wtc-perspective-card.js (hunk 1 above)
Comment (NicholasLancey) [reply src/wtc-perspective-card.js]: Good spot, removed!

**Comment by andrewrubin** [comment]
Noice, merging and publishing 🔥

**Pull request merged**